DC-SLES-SAP-sol-monitoring

```diff
@@ -1,5 +1,5 @@
 MAIN="book_solution_based.xml"
-ROOTID="art-sol-monitoring"
+ROOTID="article-sol-monitoring"
```

Comment:
maybe we can leave out the 'sol' (for solution here) and call the root ID 'article-sap-monitoring' (which would also be closer to the title?)